geniza/common/solr.py

```diff
@@ -0,0 +1,19 @@
+from parasolr import schema
+
+
+class SolrSchema(schema.SolrSchema):
+    '''Solr Schema declaration.'''
+
+    item_type = schema.SolrStringField()
+
+    # have solr automatically track last index time
+    last_modified = schema.SolrField('pdate', default='NOW')
```

Comment:
Hmm, interesting point. We've used this convention before to provide a more comprehensive and easily accessible last-modified header for requests on public pages, even though it's true that technically it's a last indexed.  I'm inclined to keep it named that way since I expect we'll use it for the same purpose. @thatbudakguy any opinion?

Reply:
I don't remember tracking this in solr before...if we're using it for sending `last-modified` headers, then I think the only place we need it in solr is in search results so that we could accurately send the most recent modification date for the whole set of results. If we do that, it still seems like we'd want the database date. @rlskoeser can you link an example of how we used it before?

Reply:
OK cool, glad to see we already wrote this mixin – this is the use case I was thinking about. I'm fine with tracking it in solr so we can reuse this code. We will end up using the database version on detail views though, right? No point in making a call to solr there (Wagtail does this automatically for `Page` subtypes, I think, but obviously not for django views)

Reply:
I think I would probably use solr last modified for all http views (because it's more inclusive / volatile), but would lean towards using the database last-modified in data downloads (because it's more conservative / less volatile).  We can always rename the solr field if it causes conflict later on.